pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -793,9 +793,9 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 			// repetitive log spam
 			switch {
 			case err == images.ErrImagePullBackOff:
-				klog.V(3).Infof("%v start failed: %v: %s", typeName, err, msg)
+				klog.V(3).Infof("%v %q start failed in pod %q: %v: %s", typeName, spec.container.Name, format.Pod(pod), err, msg)
```

Comment:
not part of this PR, but would it be nice to customize the log message instead of `%v` for the known `err` object? Would be more human friendly

Reply:
Sure, what do you have in mind?

Looking at this code, I think all this is doing is changing the log level if it is an image pull backoff error, so it doesn't flood the log with error messages.